docs/docs/guides/advanced/support_additional_rpc_methods/index.md

```diff
@@ -23,7 +26,27 @@ Following tutorial will guide you through the process of creating a custom plugi
 This will give your plugin access to [requestManager](/api/web3-core/class/Web3Context#requestManager) and [accountProvider](/api/web3-core/class/Web3Context#accountProvider).
 :::caution
 
-```ts
+
+<Tabs groupId="prog-lang" queryString>
+
+  <TabItem value="javascript" label="JavaScript"
+  	attributes={{className: "javascript-tab"}}>
+
+```javascript
+const { Web3PluginBase } = require('web3');
+
+export default class CustomRpcMethodsPlugin extends Web3PluginBase {
```

Comment:
This js code is mixture of es and cjs and it will result in error!

using it CJS , there is :
```
export class CustomRpcMethodsPlugin extends Web3PluginBase {
^^^^^^

SyntaxError: Unexpected token 'export'
```
using esm:
```
const { Web3PluginBase } = require('web3');
                           ^
ReferenceError: require is not defined in ES module scope, you can use import instead
```

Reply:
Sorry for the mess. It is corrected now.